### PR TITLE
[okina] Feedback updates [dmed256:okina]

### PR DESCRIPTION
**Pull request by dmed256**
A lot of the changes came from `make style`, the rest tried to address some Okina PR comments

### CODE REVIEW

**Comment by tzanio** [comment]
Is this ready for review?

**Comment by dmed256** [comment]
@tzanio Yep

File: mesh/mesh.cpp [hunk 1]
```diff
@@ -2165,14 +2165,14 @@ void Mesh::Make3D(int nx, int ny, int nz, Element::Type type,
       {
          for (x = 0; x < nx; x++)
          {
-            ind[0] = VTX(x  , y  , z  );
-            ind[1] = VTX(x+1, y  , z  );
+            ind[0] = VTX(x, y, z  );
+            ind[1] = VTX(x+1, y, z  );
             ind[2] = VTX(x+1, y+1, z  );
```
Comment (tzanio) [inline mesh/mesh.cpp]: Nope

File: linalg/dtensor.hpp [hunk 2]
```diff
@@ -43,9 +43,7 @@ class TensorInd
 public:
    MFEM_HOST_DEVICE static int result(const int* sizes, T first, Args... args)
    {
-#ifndef MFEM_USE_CUDA
```
Comment (camierjs) [inline linalg/dtensor.hpp]: I think we do need these if**n**def-s to avoid calling the `MFEM_ASSERT` host function.

Comment (dmed256) [reply linalg/dtensor.hpp]: Oops, added them back!

**Pull request merged**